newsroom/topics/topics.py

```diff
@@ -65,6 +65,7 @@ class TopicsResource(newsroom.Resource):
     datasource = {"source": "topics", "default_sort": [("label", 1)]}
     allowed_roles = [role for role in UserRole]
     allowed_item_roles = allowed_roles
+    internal_resource = True
```

Comment:
Is the old topics resource required anymore?

Reply:
yes, untill we not convert reports to async

Reply:
Can you place a comment at the top of this line, with prefix TODO-ASYNC, keeping around until reports are updated to async

Reply:
sure